src/theme/base/_variables.scss

```diff
@@ -1,4 +1,9 @@
+@import url('https://fonts.googleapis.com/css?family=Fjalla+One');
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700');
+
```

Comment:
I think this is not need anymore as we are using [webfontloader](https://github.com/typekit/webfontloader) which had been added in PR #67 and can be found in [/src/index.js](https://github.com/Coding-Coach/coding-coach/blob/development/src/index.js). As the application grows using import is going to be a render blocking call, and might affect page load on slow networks.

Reply:
Yes, this is not needed any more